x/consortium/spec/03_events.md

```diff
@@ -0,0 +1,7 @@
+<!--
+order: 3
+-->
+
+# Events
+
+The consortium module emits proto events defined in the Protobuf reference. TODO: use buf schema registry.
```

Comment:
We should consider using buf schema registry for the protobufs.
Introduction: https://docs.buf.build/
cosmos: https://buf.build/cosmos

Reply:
If we use buf schema, is the events of this consortium module added in this file automatically?

Reply:
No, I have to add the reference manually, after merging the corresponding PR of #491.